pkg/apiserver/handlers/featuregates/handler.go

```diff
@@ -74,8 +68,16 @@ func HandleFunc(k8sclient clientset.Interface) http.HandlerFunc {
 			return
 		}
 
-		agentfeatureGates := getAgentGatesResponse(agentConfig)
-		controllerfeatureGates := getControllerGatesResponse()
+		controllerConfig := &Config{}
+		err = yaml.Unmarshal([]byte(antreaConfig.Data["antrea-controller.conf"]), controllerConfig)
```

Comment:
Please define a const for `"antrea-controller.conf"`, so does `"antrea-agent.conf"`.

Reply:
fix: https://github.com/antrea-io/antrea/pull/5314/commits/fc10e28d0a95dce6c74926002014d93fd597e640